application/src/main/java/org/opentripplanner/standalone/configure/ConstructApplicationModule.java

```diff
@@ -52,27 +53,35 @@ OtpServerRequestContext providesServerContext(
   ) {
     var defaultRequest = launcherRequestDecorator.intercept(routerConfig.routingRequestDefaults());
 
-    return DefaultServerRequestContext.create(
-      routerConfig.transitTuningConfig(),
-      defaultRequest,
-      raptorConfig,
+    var transitRoutingConfig = routerConfig.transitTuningConfig();
+    var vectorTileConfig = routerConfig.vectorTileConfig();
+    var flexParameters = routerConfig.flexParameters();
+
+    // TODO: Inject this, can this use the routerConfig routingRequest ?
+    var tileRendererManager = new TileRendererManager(graph, defaultRequest.preferences());
```

Comment:
This is moved up from the `DefaultServerRequestContext`, the injection should be done by Dagger.